docs/decisions/0001-record-architecture-decisions.md

```diff
@@ -0,0 +1,19 @@
+# 1. Record architecture decisions
```

Comment:
Should/could we move https://github.com/dimagi/commcare-cloud/tree/master/decisions in here or remove it so there aren't two places for decisions?

Reply:
ha, I guess I missed that one